Add instructions for `apt-get` install to `README`

ldennington opened the pull request:
Updating `README.md` with instructions for `apt-get` setup and install
for Ubuntu Bionic and Hirsute.

Commit:
Add instructions for `apt-get` install to `README`

Updating `README.md` with instructions for `apt-get` setup and install
for Ubuntu Bionic + Hirsute.

derrickstolee commented on README.md:
```diff
@@ -112,9 +112,33 @@ Or you can run the `git update-microsoft-git` command, which will run those brew
+Hippo (21.04). Take the following steps to set up and install based on the
+version you are running:
+
+### Bionic
```
Perhaps list this subsection as

```suggestion
### Ubuntu 18.04 (Bionic)
```

Similarly, `Ubuntu 21.04 (Hirsute)` in the one below.

derrickstolee commented on README.md:
```diff
@@ -112,9 +112,33 @@ Or you can run the `git update-microsoft-git` command, which will run those brew
+curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
+sudo apt-add-repository https://packages.microsoft.com/ubuntu/18.04/prod
+sudo apt-get update
+sudo apt-get install microsoft-git
```
Did we update the action to publish as `microsoft-git`, or is it still `git-vfs`? Can that be part of this PR?

ldennington replied:
No updates to the pipeline are required to publish under a new name. However, there is one small clarifying tweak I want to make, which I will include as a separate commit in this PR. Stay tuned!

ldennington committed:
Clarify `workflow_dispatch` input description

We are currently using 'Release tag' to describe the required input
to our `workflow_dispatch` trigger. This is inaccurate - this field
actually requires a 'Release id', which I discovered when testing
GCM Core `apt-get` deployments yesterday. Updating so that the
description doesn't confuse folks running the workflow for a release
that is not 'latest'.

derrickstolee reviewed (approved):
LGTM. [Pushing the `microsoft-git` version now](https://github.com/microsoft/git/actions/runs/990944865).
